lib/oli_web/templates/shared/_help_button.html.eex

```diff
@@ -0,0 +1,4 @@
+<a class="btn btn-xs btn-light rounded-pill help-btn mx-2" href='javascript:;' onclick="showHelpModal();">
```

Comment:
if it looks like a button, and it acts like a button, maybe it should be a `<button>`?